### DISCUSSION TITLE
Multiplayer games: TCP / IPX / Modem

### DISCUSSION THREAD

**Discussion by kcgen** [dosbox-staging/dosbox-staging#1512] [General]
Feel free to add games, their communication method(s), patches or versions (as sometimes multiplayer was fixed in later versions), and if you've tested them emulators or on real hardware.

**Comment by 77ZaRR77** [comment]
These are most of my interest:
3-D Helicopter Simulator
Advanced Netwars
Alien Trilogy (demo too)
Armor Alley (modem)
BMF
Carmageddon
Chasm The Rift
DOG Fight for your life
Darkflight Conflict
Dungeon Keeper 1
Eradicator
Extreme Assault
Fire Power
GTA 1
Gemini-2
International Moto X
Last Rites
Mass Destruction
Modem Wars
StrikePoint: The Hex Missions
Terminal Velocity
Terminator SkyNET
Track Attack
The Need for Speed: Special Edition
Theme Hospital (apply patch)
Vette! (modem)
Z.A.R
Берегись автомобиля! (modem) (Beregis avtomobilya)
